awx_collection/tools/templates/tower_module.j2

```diff
@@ -165,7 +194,7 @@ def main():
 {% if option == name_option %}
     new_fields['{{ name_option }}'] = new_{{ name_option }} if new_{{ name_option }} else {{ name_option }}
 {% else %}
-    if {{ option }}:
+    if {{ option }} != None:
```

Comment:
don't you need to use `is not None` for the linter to pass?

Reply:
Yes, you're correct.  I'll edit the syntax for these conditional statements.